### ISSUE TITLE
Model Selection

### ISSUE THREAD

**Issue by ottaviofornieri** [ottaviofornieri/HandWrittenDigitRecognition#1]
I want to add model selection, implementing different neural networks, namely different sets of layers with different units each.
Now that the building/training of the neural network is inside a class, I can choose among different models. **Use a cross-validation set.**
